app/src/app/[lng]/onboarding/setup/page.tsx

```diff
@@ -1,75 +1,34 @@
 "use client";
 
-import RecentSearches from "@/components/recent-searches";
 import WizardSteps from "@/components/wizard-steps";
-import { set } from "@/features/city/openclimateCitySlice";
 import { useTranslation } from "@/i18n/client";
-import { useAppDispatch } from "@/lib/hooks";
 import type { CityAttributes } from "@/models/City";
 import {
   api,
   useAddCityMutation,
   useAddCityPopulationMutation,
   useAddInventoryMutation,
-  useGetOCCityDataQuery,
-  useGetOCCityQuery,
   useSetUserInfoMutation,
 } from "@/services/api";
-import {
-  findClosestYear,
-  getShortenNumberUnit,
-  shortenNumber,
-} from "@/util/helpers";
+
 import { OCCityAttributes } from "@/util/types";
-import {
-  ArrowBackIcon,
-  CheckIcon,
-  InfoOutlineIcon,
-  SearchIcon,
-} from "@chakra-ui/icons";
-import { Link } from "@chakra-ui/next-js";
-import {
-  Box,
-  Button,
-  Card,
-  Flex,
-  FormControl,
-  FormErrorIcon,
-  FormErrorMessage,
-  FormLabel,
-  HStack,
-  Heading,
-  Icon,
-  Input,
-  InputGroup,
-  InputLeftElement,
-  InputRightElement,
-  Select,
-  Text,
-  useOutsideClick,
-  useSteps,
-  useToast,
-} from "@chakra-ui/react";
-import type { TFunction } from "i18next";
-import dynamic from "next/dynamic";
+import { ArrowBackIcon, ArrowForwardIcon } from "@chakra-ui/icons";
+import { Box, Button, Text, useSteps, useToast } from "@chakra-ui/react";
+
 import { useRouter } from "next/navigation";
-import { useEffect, useRef, useState } from "react";
-import type {
-  Control,
-  FieldErrors,
-  SubmitHandler,
-  UseFormRegister,
-} from "react-hook-form";
+import { useState } from "react";
+import type { SubmitHandler } from "react-hook-form";
 import { useForm } from "react-hook-form";
-import { Trans } from "react-i18next/TransWithoutContext";
-import { MdOutlineAspectRatio, MdOutlinePeopleAlt } from "react-icons/md";
-import FormattedThousandsNumberInput from "@/app/[lng]/onboarding/setup/FormattedThousandsNumberInput";
+import SelectCityStep from "../../../../components/steps/select-city-steps";
+import SetInventoryDetailsStep from "../../../../components/steps/add-inventory-details-step";
+import SetPopulationDataStep from "../../../../components/steps/add-population-data-step";
+import ConfirmStep from "../../../../components/steps/confirm-city-data-step";
```

Comment:
Can you use the `@/app` alias we have instead of these long relative paths? Alternatively you can also move these components into the page's folder like we are doing for the manual input pages.